src/dmd/mtype.d

```diff
@@ -4517,11 +4525,12 @@ extern (C++) final class TypeFunction : TypeNext
             tf.isproperty = t.isproperty;
             tf.isref = t.isref;
             tf.isreturn = t.isreturn;
-            tf.isscope = t.isscope;
+            tf.isthisscope = t.isthisscope;
             tf.isreturninferred = t.isreturninferred;
             tf.isscopeinferred = t.isscopeinferred;
             tf.trust = t.trust;
-            tf.iswild = t.iswild;
+            tf.isInOutParam = t.isInOutParam;
+            tf.isInOutQual = t.isInOutQual;
```

Comment:
Unrelated change?

Reply:
previously the two values were packaed in the `iswild` byte. now they are part of the new flags.